### DISCUSSION TITLE
Outline integration with next.js and Ghost CMS

### DISCUSSION THREAD

**Discussion by junevm** [outline/outline#3026] [Ideas]
Hi first have a look at this https://samuelkraft.com/blog/building-a-notion-blog-with-public-api

Can this functionality of adding/editing pages in outline and the changes being reflected in next.js site or Ghost cms blog be achieved.

It yes can you make an integration with ghost cms. It would be very helpful in writing blogs without leaving outline

**Comment by tommoor** [comment]
Someone could definitely build it, seems like it would be relatively simple as far as things go :)

**Comment by tennox** [comment]
I think gridsome with a [REST source](https://www.npmjs.com/package/gridsome-source-rest) configured for [Outline API](https://www.getoutline.com/developers#tag/Documents/paths/~1documents.info/post) could result in a good PoC ;)
